packages/gestalt/src/TapArea.js

```diff
@@ -116,7 +121,6 @@ function TapArea({
         if (!disabled && onFocus) {
           onFocus({ event });
         }
-        event.stopPropagation();
```

Comment:
@jackhsu978 could you verify that this is ok to remove? You added it back in #906 for the following reason:

> ### Why stops onFocus event from bubbling up?
> See diff D304752
>> "This makes it so that with the STL ... on a pin is focused the whole pin is not focused."

We do need the focus event to bubble up in this case in order for `useFocusVisible` to work properly. Now that we only show focus styles on keyboard navigation, I hope this is ok.

Reply:
I think the approach sounds good. The side effect here is not too implicit.